manifests/pip.pp

```diff
@@ -220,7 +220,7 @@
 
       default: {
         # Anti-action, uninstall.
-        $command        = "echo y | ${pip_env} uninstall ${uninstall_args} ${proxy_flag} ${name}"
+        $command        = "echo y | ${pip_env} uninstall ${uninstall_args} ${proxy_flag} ${real_pkgname}"
```

Comment:
thanks for the PR! Can you take a look at the acceptance tests in spec/acceptance/ and add one there that reflects this change? If you need any help with it feel free to join our IRC channel #voxpupuli on Freenode our or slack channel #voxpupuli on https://slack.puppet.com/